test/docs/printer_code_test.py

```diff
@@ -1,7 +1,54 @@
+from collections import namedtuple
+from pytest import fixture
 from sympy import Integral, evaluate, exp, pi, sin, sqrt
-from symplyphysics import SymbolNew, clone_as_symbol, quantities, symbols, clone_as_function
+from symplyphysics import SymbolNew, Quantity, clone_as_symbol, clone_as_function
 from symplyphysics.docs.printer_code import code_str
 
+Args = namedtuple(
+    "Args",
+    [
+        "mass",
+        "temperature",
+        "boltzmann_constant",
+        "energy",
+        "time",
+        "force",
+        "speed",
+        "speed_of_light",
+        "charge",
+        "distance",
+        "vacuum_permittivity",
+    ],
+)
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    mass = SymbolNew("m")
```

Comment:
Better take them from symbols.

Reply:
The issue was that I changed the representation of a symbol and it crashed. I'm not sure it's the best thing to go and check all test strings when something in `symbols` gets changed.